packages/embed/src/factories/create-sidetab/sidetab-options.ts

```diff
@@ -0,0 +1,18 @@
+import { ActionableOptions, UrlOptions, BehavioralOptions } from '../../base'
+
+export type SidetabOptions = UrlOptions &
+  ActionableOptions &
+  BehavioralOptions & {
+    /**
+     * Specify the button text
+     *
+     * @type {string}
+     */
+    buttonText?: string
+    /**
+     * Specify the button background color
+     *
+     * @type {string}
+     */
+    buttonColor?: string
```

Comment:
We should also load those options from attributes in [loadOptionsFromAttributes](https://github.com/Typeform/embed/blob/6d7c673b0694318525c5b9774f885c319f29eb10/packages/embed/src/utils/load-options-from-attributes.ts#L58).